### ISSUE TITLE
Accessibility: MAS05A: Keyboard: keyboard focus is not visible for header menu items. 

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#30]
#A11yMAS
#MAS05A
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/ in edge.
2.Navigate to menu items "Docs","Updates","Blog","Extensions","FAQ" beside the "visual studio code" in header of the page.
3.Observe the keyboard focus.

**Actual Result:**
Keyboard focus is not visible for "Docs","Updates","Blog","Extensions","FAQ"  menu items in header of the page.

**Expected Result:**
keyboard focus should be visible for all the interactive elements in the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may not be able to identify that the element is focused or not.

**Screenshots:**
![inspect view_menu items](https://cloud.githubusercontent.com/assets/24825441/21709693/1bbe202c-d409-11e6-9281-dc0d9a702611.png)
![no visual focus for header items](https://cloud.githubusercontent.com/assets/24825441/21709697/21bf2642-d409-11e6-81c2-a256c026cb61.png)

**Comment by waderyan** [comment]
Can't reproduce. 

![image](https://cloud.githubusercontent.com/assets/2449568/26378841/20ba4aa0-3fcb-11e7-89d9-08797f737b56.png)
